### PR TITLE
Add support to specify CASCADE when dropping views

### PR DESCRIPTION
**Pull request by amicks**
I ran into an issue with SQLite where views would be dropped, resulting in a SQLAlchemy.exc.OperationalError for syntax on the "CASCADE" suffix.

Some SQL DBMSs like MSSQL or SQLite don't support the CASCADE suffix so the caller should be able to specify this.  Otherwise, dropping views is not possible on those DBMSs.

### CODE REVIEW

File: sqlalchemy_utils/view.py [hunk 1]
```diff
@@ -149,7 +152,11 @@ def create_indexes(target, connection, **kw):
         for idx in table.indexes:
             idx.create(connection)
 
-    sa.event.listen(metadata, 'before_drop', DropView(name))
+    sa.event.listen(
```
Comment (amicks) [inline sqlalchemy_utils/view.py]: This change isn't functional and is only because CI was failing on the lint job for too many characters in one row.

**Comment by amicks** [comment]
CI build is failing on unrelated code in encryption TestAesGcmEngine.

**Comment by amicks** [comment]
Build is passing now :)

**Comment by kvesteri** [comment]
Thanks for the PR! 🚋

**Pull request merged**